packages/fxa-auth-server/lib/senders/emails/templates/subscriptionAccountFinishSetup/index.stories.ts

```diff
@@ -0,0 +1,40 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Meta } from '@storybook/html';
+import { subplatStoryWithProps } from '../../storybook-email';
+
+export default {
+  title: 'SubPlat Emails/Templates/subscriptionAccountFinishSetup',
+} as Meta;
+
+const createStory = subplatStoryWithProps(
+  'subscriptionAccountFinishSetup',
+  'Sent to a user after they purchased the product through the password-less flow without an existing Firefox account.',
+  {
+    productName: 'Firefox Fortress',
+    icon: 'https://accounts-static.cdn.mozilla.net/product-icons/mozilla-vpn-email.png',
+    link: 'http://localhost:3030/post_verify/finish_account_setup/set_password',
+    subscriptionSupportUrl: 'http://localhost:3030/support',
+  }
+);
+
+export const SubscriptionFirstInvoiceMissingDetails = createStory(
+  {
+    invoiceDateOnly: '10/13/2021',
+    invoiceNumber: '8675309',
+    invoiceTotal: '$20.00',
+  },
```

Comment:
I think all of these can actually be missing, right? I like calling this "missing details," but I think it should show what it looks like when all possible missing details are missing. If the template should always have these variables, we shouldn't need the conditional for them.

Unless this is quick, I'm OK with exploring that in a follow up (in the bugs & tasks epic) as a 1-2 pointer to not block here because if those 3 can actually be null/undefined in practice we should update this story to show that, and if they can't, we should modify the template file and take out conditionally displaying them.

Reply:
Good call, I'm aligned with this and will add these stories in!